locale/en/blog/release/v0.8.0.md

```diff
@@ -96,14 +96,12 @@ Requests per second:    3317.24 [#/sec] (mean)
 # 0.8.0
 Requests per second:    3795.34 [#/sec] (mean) +14.41%
 
-
```

Comment:
This changes line in `pre`s too.

Reply:
Good catch!
Looking at the other blocks in this file, the double line in the other calls isn't consistent, so i don't think it matters in this case